flutter/ios/Classes/SentryFlutterPluginApple.swift

```diff
@@ -11,6 +11,7 @@ import AppKit
 public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
 
     private var sentryOptions: Options?
+    let sentryIOSPrivate = PrivateSentrySDKOnly()
```

Comment:
We can either replace all the calls with `sentryIOSPrivate` now or just use `PrivateSentrySDKOnly` since it's a static class.

Reply:
the method we need is not static: 

https://github.com/getsentry/sentry-cocoa/blob/17969fa1c80d643df62aa92e8a5fa724c9af0d8c/Sources/Sentry/Public/PrivateSentrySDKOnly.h#L42

Reply:
Oh I see, maybe it was a mistake on the Cocoa SDK, let's double-check this.
@brustolin or @philipphofmann is there a reason why `getDebugImages` isn't static as every other method?

Reply:
I believe is because, `getDebugImages` need a `SentryDebugImageProvider` instance and we don't want to create a new one every call. But I guess we could move the provider to `SentryDependencyContainer` and make the Hybrid class full static.

Reply:
already resolved in https://github.com/getsentry/sentry-cocoa/pull/1769